src/m3ninx/search/query/codec.go

```diff
@@ -85,8 +88,7 @@ func unmarshal(q *querypb.Query) (search.Query, error) {
 			qs = append(qs, sqry)
 		}
 		return NewDisjunctionQuery(qs), nil
-
 	}
 
-	return nil, fmt.Errorf("unknown query: %v", q)
+	return nil, fmt.Errorf("unknown query: %+v", q)
```

Comment:
Wow, didn't know about this, it's pretty neat :+1:

Reply:
lol :)